Description: Use FHS compliant path for data
 By default, mercurial installs expects its data files next to their
 py files in /usr/lib/python2.7/dist-packages/mercurial .  Debian puts
 the data files in the FHS compliant /usr/share/mercurial .  Fix the
 path only when run from the FHS compliant path
 (/usr/lib/pythonX.Y/dist-packages/mercurial) so as to not break the
 testsuite when building the package.
Author: Javi Merino <vicho@debian.org>

Index: mercurial-4.4.1/mercurial/util.py
===================================================================
--- mercurial-4.4.1.orig/mercurial/util.py	2017-11-13 07:05:09.546415215 +0200
+++ mercurial-4.4.1/mercurial/util.py	2017-11-13 07:05:09.542415144 +0200
@@ -1094,6 +1094,8 @@
 if mainfrozen() and getattr(sys, 'frozen', None) != 'macosx_app':
     # executable version (py2exe) doesn't support __file__
     datapath = os.path.dirname(pycompat.sysexecutable)
+elif __file__.startswith("/usr/lib/python"):
+    datapath = "/usr/share/mercurial"
 else:
     datapath = os.path.dirname(pycompat.fsencode(__file__))
 
